### PR TITLE
Add Facebook Login

### PR DESCRIPTION
**Pull request by alvisanovari**
### WHAT?!

Added Facebook login in Social media logins.

### CODE REVIEW

File: gatsby-theme-firebase/src/components/SocialLogins.tsx [hunk 1]
```diff
@@ -63,6 +65,21 @@ const SocialLogins: React.FunctionComponent<{
           <TwitterIcon sx={{ mr: 1 }} size={18} /> Sign in with Twitter
         </SocialLoginButton>
       )}
+      {enableFacebook && (
```
Comment (epilande) [inline gatsby-theme-firebase/src/components/SocialLogins.tsx]: This variable isn't defined. 

Add this on line#26:
```jsx
  const enableFacebook = socialLogins.includes("facebook");
```

File: gatsby-theme-firebase/src/components/icons/Facebook.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    const aspectRatio = 98 / 120;
+    const height = size * aspectRatio;
+    return (
+        <svg width={size} height={height} viewBox="0 0 120 98" {...restProps}>
+            <title>Faceboook</title>
+            <g fill="none" fillRule="evenodd">
+                <path d="M0-13h120v120H0V-13z" />
+                <path
+                    d="M40,0H10C4.486,0,0,4.486,0,10v30c0,5.514,4.486,10,10,10h30c5.514,0,10-4.486,10-10V10C50,4.486,45.514,0,40,0z M39,17h-3 c-2.145,0-3,0.504-3,2v3h6l-1,6h-5v20h-7V28h-3v-6h3v-3c0-4.677,1.581-8,7-8c2.902,0,6,1,6,1V17z"
+                    fill="#55acee"
```
Comment (epilande) [inline gatsby-theme-firebase/src/components/icons/Facebook.tsx]: Probably want to change the fill color to FB blue i.e. `#3A5C98`

File: gatsby-theme-firebase/src/components/icons/Facebook.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+const Faceboook: React.FunctionComponent<Props> = ({
+    size = 16,
+    ...restProps
+}) => {
+    const aspectRatio = 98 / 120;
+    const height = size * aspectRatio;
+    return (
+        <svg width={size} height={height} viewBox="0 0 120 98" {...restProps}>
```
Comment (epilande) [inline gatsby-theme-firebase/src/components/icons/Facebook.tsx]: Let's fix the svg size. We can remove `aspectRatio` and `height`.
And we can update the viewBox to `0 0 50 50`. 

```diff
-    const aspectRatio = 98 / 120;
-    const height = size * aspectRatio;
     return (
-        <svg width={size} height={height} viewBox="0 0 120 98" {...restProps}>
+        <svg width={size} height={size} viewBox="0 0 50 50" {...restProps}>
```

File: gatsby-theme-firebase/src/components/icons/Facebook.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            <title>Faceboook</title>
+            <g fill="none" fillRule="evenodd">
+                <path d="M0-13h120v120H0V-13z" />
```
Comment (epilande) [inline gatsby-theme-firebase/src/components/icons/Facebook.tsx]: This looks like it can also be removed.

```diff
-            <path d="M0-13h120v120H0V-13z" />
```

File: demo/gatsby-config.js [hunk 5]
```diff
@@ -22,7 +22,7 @@ module.exports = {
           appId: process.env.FIREBASE_APP_ID,
         },
         loginPath: "/login",
-        socialLogins: ["google", "github"],
+        socialLogins: ["google", "github", "facebook"],
```
Comment (epilande) [inline demo/gatsby-config.js]: Let's not add new socialLogins to the demo. You can do this locally, but please don't commit it.

**Comment by alvisanovari** [comment]
Thanks for the feedback! made the changes.

**Pull request merged**